logger/consoleFormatter.go

```diff
@@ -45,7 +61,7 @@ func formatArgs(levelColor string, args ...string) string {
 
 	argString := ""
 	for index := 1; index < len(args); index += 2 {
-		argString += fmt.Sprintf("\033[%s%s\033[0m=%s ", levelColor, args[index-1], args[index])
+		argString += fmt.Sprintf("\033[%s%s\033[0m = %s ", levelColor, args[index-1], args[index])
```

Comment:
The original version made "empty" values more visible, because you would expect the value to start immediately after `=`. Would rather have the original.

Reply:
@bogdan-rosianu @iulianpascalau will decide this :)

Reply:
I suggested this change (actually did this change) for a better experience when selecting a value (double-clicking the value now selects `key=value` but with this change it will select only the `value`).  But this is still debatable, we might wait for other opinions